modules/ROOT/pages/server_release_notes.adoc

```diff
@@ -40,9 +41,17 @@
 * xref:changes-in-8-0[Changes in 8.0]
 * xref:changes-in-7-0[Changes in 7.0]
 
+== Changes in 10.12.2
+
+Dear ownCloud administrator, find below the changes and known issues in ownCloud Server 10.12.2 that need your attention. You can also read the {owncloud-server-changelog-url}[full ownCloud Server changelog] for further details on what has changed.
+
+=== Filter sensitive data in log for Session::loginInOwnCloud
+
+The `loginInOwnCloud` method contains sensitive data in the argument list and needed therefore to be added to the list of methods where sensitive parameters are being obfuscated. https://github.com/owncloud/core/pull/40792[#40792]
+
 == Changes in 10.12.1
 
-Dear ownCloud administrator, find below the changes and known issues in ownCloud Server 10.12.1 that need your attention. You can also read the {oc-changelog-url}[full ownCloud Server changelog] for further details on what has changed.
+Dear ownCloud administrator, find below the changes and known issues in ownCloud Server 10.12.1 that need your attention. You can also read the {owncloud-server-changelog-url}[full ownCloud Server changelog] for further details on what has changed.
```

Comment:
```suggestion
Dear ownCloud administrator, find below the changes and known issues in ownCloud Server 10.12.1 that need your attention. You can also read the {oc-changelog-url}[full ownCloud Server changelog] for further details on what has changed.
```
There is no `owncloud-server-changelog-url` attribute, see `site.yml`

Reply:
Before those changes links were not not correctly displayed. There seems to be `:owncloud-server-changelog-url: https://owncloud.com/changelog/server/` in modules/ROOT/pages/server_release_notes.adoc ?

Reply:
Ah apparently links in the online release notes are displayed correctly. Will revert the changes.